### PR TITLE
Adding celery 4.1.0, new icon and generation instructions

### PR DESCRIPTION
**Pull request by jamesmallen**

### CODE REVIEW

File: docsets/Celery/docset.json [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+        "name": "James M. Allen",
+        "link": "https://github.com/jamesmallen"
     },
     "aliases": [],
 
     "specific_versions": [
+        {
+            "version": "4.1.0",
+            "archive": "versions/4.1.0/Celery.tgz"
+        }
```
Comment (Kapeli) [inline docsets/Celery/docset.json]: The JSON isn't valid. I think there's a missing comma at the end of this line.

**Comment by Kapeli** [comment]
The docset archive should contain only the docset and no other folders (or parent folders).

File: docsets/Celery/docset.json (hunk 1 above)
Comment (jamesmallen) [reply docsets/Celery/docset.json]: Woops - fixed!

**Comment by jamesmallen** [comment]
@Kapeli I updated the instructions in the README and the docset archives. Thanks!

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**